Cargo.toml

```diff
@@ -118,6 +118,7 @@ libp2p = { version = "0.53", default-features = false, features = [
   'tokio',
   'macros',
   'serde',
+  'rsa',
```

Comment:
Should we explicitly enable all of `secp256k1`, `ecdsa`, `rsa` and `ed25519`?

Reply:
Let's add them. I'll ask around to ensure it's ultimately what we need.

Reply:
As per the code, the list should be exhaustive
```rust
#[derive(Debug, Clone)]
#[allow(clippy::large_enum_variant)]
enum KeyPairInner {
    /// An Ed25519 keypair.
    #[cfg(feature = "ed25519")]
    Ed25519(ed25519::Keypair),
    /// An RSA keypair.
    #[cfg(all(feature = "rsa", not(target_arch = "wasm32")))]
    Rsa(rsa::Keypair),
    /// A Secp256k1 keypair.
    #[cfg(feature = "secp256k1")]
    Secp256k1(secp256k1::Keypair),
    /// An ECDSA keypair.
    #[cfg(feature = "ecdsa")]
    Ecdsa(ecdsa::Keypair),
}
```